interpreter/exec/v128.ml

```diff
@@ -2,6 +2,8 @@ include Simd.Make
   (struct
     include Bytes
```

Comment:
Ideally, the rep type should be string, too (the representation of Wasm values should always be immutable). But we can fix that later.